app/controllers/login.js

```diff
@@ -22,6 +22,7 @@ export default Ember.Controller.extend({
         .authenticate('authenticator:conduit', { email, password })
         .then(() => {
           this.transitionToRoute('home');
+          get(this, 'session').authorize('authorizer:conduit');
```

Comment:
Does `get(this, 'session').authorize()` return a promise? I'm wondering if this should return the value from here...